openshift/ci-operator/update-ci.sh

```diff
@@ -46,11 +46,13 @@ echo "Generating PROW files in $OPENSHIFT"
 make jobs
 make ci-operator-config
 # We have to do this manually, see: https://docs.ci.openshift.org/docs/how-tos/notification/
-echo "==== Adding reporter_config to periodics ===="
-# These version MUST match the ocp version we used above
-for OCP_VERSION in 47; do
-    sed -i "/  name: periodic-ci-openshift-knative-eventing-kafka-release-${VERSION}-${OCP_VERSION}-e2e-aws-ocp-${OCP_VERSION}-continuous\n  spec:/ r $TMPDIR/reporterConfig" "$PERIODIC_CONFIG"
-done
+if [[ "$VERSION" != "next" ]]; then
+  echo "==== Adding reporter_config to periodics ===="
+  # These version MUST match the ocp version we used above
+  for OCP_VERSION in 47 48; do
```

Comment:
with 4.9 and 4.10 coming, can we organize this differently?

Reply:
😄 I did create a list of versions on top of the file like
```bash
OCP_VERSIONS="4.7 4.8 4.9" 
```
etc but then that resulted in some if statements etc etc so I thought to take it to a different PRs

my understanding is that this script is being "explicit" in every version of OCP it's generating for, hence the 
```
$CURDIR/generate-ci-config.sh knative-$VERSION 4.7 > ${CONFIG}__47.yaml
$CURDIR/generate-ci-config.sh knative-$VERSION 4.8 true > ${CONFIG}__48.yaml
```

so, let's do this in a different PR to make all that generation dynamic based on a list of versions defined on top